templates/control/2fa/disable.html

```diff
@@ -0,0 +1,29 @@
+$def with (username, error)
+$:{RENDER("common/stage_title.html", ["Disable 2FA", "Two-Factor Authentication", "/control/2fa/status"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/2fa/disable">
+    $if error == "2fa":
+      <div id="signin_error"><strong>Whoops!</strong> The 2FA response token or recovery code you provided did not validate.</div>
+    $elif error == "verify":
+      <div id="signin_error"><strong>Hey!</strong> Did you want to disable 2FA? You didn't check the verification checkbox!</div>
+    <h3>Disable Two-Factor Authentication (2FA)</h3>
+    <p><strong>Username: $:{MARKDOWN_EXCERPT(username)}</strong></p>
```

Comment:
You asked to have the username escaped through the Markdown renderer. Straight ``$:{MARKDOWN(username)}`` was mucking with the formatting of the text (I'll go back and get a screencap to make sure it still presents the issue). So I tested the ``MARKDOWN_EXCERPT`` mechanism, and it wasn't causing issues with the text, so I used it.

I'll grab a screenshot shortly.